bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncAdmin.java

```diff
@@ -277,71 +320,240 @@ public BigtableAsyncAdmin(BigtableAsyncConnection asyncConnection) throws IOExce
   // ****** TO BE IMPLEMENTED [start] ******
 
   @Override
-  public CompletableFuture<Void> deleteColumnFamily(TableName arg0, byte[] arg1) {
-    throw new UnsupportedOperationException("deleteColumnFamily"); // TODO
+  public CompletableFuture<Void> deleteColumnFamily(TableName tableName, byte[] columnName) {
+    return CompletableFuture.supplyAsync(() -> {
```

Comment:
Is there a reason we need supplyAsync here?  Usually that's needed when constructing the request is complex and can throw an exception.